client/src/views/Calculate.vue

```diff
@@ -1,8 +1,8 @@
 <template>
   <header-nav-bar />
-  <div class="relative isolate overflow-hidden bg-gray-900 py-24 sm:py-32">
+  <div class="relative isolate overflow-hidden bg-gray-900 py-24 sm:py-15">
     <div
-      class="max-w-screen-xl mx-auto px-4 sm:px-6 lg:px-8 flex flex-col sm:flex-row justify-between items-center"
+      class="max-w-screen-xl px-4 sm:px-4 lg:px-4 flex flex-col sm:flex-row justify-between items-center"
     >
       <!-- Form -->
       <div class="w-full sm:w-1/2 rounded-lg">
```

Comment:
Seems to be an issue with alignment on larger screens, is this relating to the `sm:` part of the tailwind?
![image](https://github.com/tomstrong64/MyJourneyFootprint-COM617/assets/128226866/bea45c88-5cbd-4e39-8ea6-99adabcce628)